fmn/rules/ci.py

```diff
@@ -0,0 +1,234 @@
+from fmn.lib.hinting import hint, prefixed as _
+
+# Basic building steps
+
+@hint(categories=['ci'])
+def ci_catchall(config, message):
+    """ All CI events
+
+    Adding this rule will indiscriminately match notifications of all types
+    from `ci <https://admin.fedoraproject.org/pkgdb>`_, i.e. new build queued,
```

Comment:
Yeah, I wasn't sure what would be a good URL to use, the wiki page? the actual jenkins instance?

Maybe the wiki page is better since the jenkins instance will be linked to in the messages.